report/2.Analysis/ArduinoInEducation.tex

```diff
@@ -0,0 +1,50 @@
+% point of document
+% We want to explain to the reader that (and how) the arduino platform is used in HTX
+% The point is:
+% - What "teknikfag" is
+% - What "el-teknik" is
+% 
+
+\section{Arduino and their usage in danish education}
+During the third year of the HTX\footnote{Higher Technical Examination Programme, is a 3-year vocationally oriented general upper secondary programme which builds on the 10th-11th form of the Folkeskole\cite{htx_wiki}} programme students are required to take a course in a voluntary technical subject, which utilizes a project-based learning practive. 
+There are a multiple of courses, one of these regard electronics, where many schools choose to utilize the Arduino platform\footnote{based on the fact that 4 of the students in this group studied at HTX in different parts of Denmark and all utilized the Arduino platform in this course}\cite{holstebro_education}.  
+
+This technical oriented course, often called electronic\footnote{The name varies between institutions}, aims to familiarize the students with electrical components and circuits as well as basic programming.\cite{holstebro_electronic}
+Arduino is an ideal platform for this, as it is possible to connect I/O devices to the Arduino micro-controller, which in turn can be programmed to do simple tasks. 
+This course is then used to both spike the interest of the student regarding engineering fields, such as programming or electronics.
+
+\subsection{HTX CAO Electronic Course}
+In relation to the discipline of understanding the initial problem. The project group contacted one of the major HTX institutions in North Jutland, Denmark. Namely HTX CAO. The following subsection will summarize the important and noticable aspects of the electronic course conducted on HTX CAO. The conclusions reached in this subsection are purely based on the statements made by the students and the teacher of this specific electronics course.
```

Comment:
Be better at new lining for one. First dot, should be a comma. Dot after Denmark, should be comma. Noticable should be notable. Do we mention what CAO means?

Reply:
Well CAO is a name